idea/api-gateway/src/rabbitmq/types/index.ts

```diff
@@ -27,5 +22,20 @@ export type KafkaParams =
   | GetOutgoingMessagesParams
   | GetTestBalanceParams
   | GetAllUserProgramsParams
-  | NetworkKafkaPartition
   | GetCodeParams
+
+interface IMessageNetworkDSParams {
```

Comment:
What is `Network`?

Reply:
This is name appeared from `isNetworkDataAvailable`. Maybe better to rename `IMessageDataStorageParams` ?

Reply:
Yeah, i guess it will be better